frontend/src/lib/candidate/components/logoutButton/LogoutButton.svelte

```diff
@@ -59,24 +63,31 @@
 Allows user to log out. Displays modal notification if the user
 hasn't filled all the data. 
 
-This component has optional boolean property `variantIcon`:
-When set to true (default), the button variant is icon. 
-When set to false, the button variant is ghost.
+This component has optional string property `buttonVariant`:
+When set to normal (default), the button variant is normal which is just plain text. 
+When set to icon, the button variant is icon.
+When set to main, the button variant is main which is a red box.
 
 ### Usage
 ```tsx
   <LogoutButton />
 ```
 -->
```

Comment:
There are also missing props in the component documentation so those should be added.
Also, the component documentation could be improved to be more in the style of the other components where the properties are clearly documented. Here is an example regarding how the properties should be documented to get a better idea:
```
### Properties

- `name`: the name of the icon to use
- `size`: The size of the icon as one of the predefined sizes 'sm', 'md' or 'lg'.
   For arbitrary values, you can supply a `class` property, such as 
   `h-[3.15rem] w-[3.15rem]`. @default 'md'
- `color`: The color of the icon as one of the predefined colours.
   For arbitrary values, you can supply a `class` property, such as
   `fill-[#123456]`. @default 'current'
- `aria-hidden`: @default `true`
- `role`: Aria role @default `img`
- `class`: Additional class string to append to the element's default classes.
```

(from `Icon.svelte`)